pkg/controller/v1beta1/inferenceservice/components/predictor.go

```diff
@@ -330,6 +332,8 @@ func (p *Predictor) reconcileModel(ctx context.Context, isvc *v1beta1.InferenceS
 		// Get first supporting runtime.
 		sRuntime = runtimes[0].Spec
 		isvc.Spec.Predictor.Model.Runtime = &runtimes[0].Name
+		// Set the ServingRuntimeName in the status
+		isvc.Status.ServingRuntimeName = runtimes[0].Name
```

Comment:
AFAIK, it is possible to have a cluster-scoped `ClusterServingRuntime` with same name as a namespace-scoped `ServingRuntime`. I think you need a way to distinguish them.